docs/src/references/tuning/base_classes.rst

```diff
@@ -0,0 +1,15 @@
+Base Classes
+############
+
```

Comment:
Maybe put the design of these classes here instead in the truning 'landing' page,